src/ceph-disk/ceph_disk/main.py

```diff
@@ -1898,10 +1898,6 @@ def parser():
             help='unique OSD uuid to assign this disk to',
         )
         parser.add_argument(
-            '--crush-device-class',
-            help='crush device class to assign this disk to',
-        )
-        parser.add_argument(
```

Comment:
I don't think we shoudl deprecate this.. we can still let the admin manually specify a different class of their choice.  Not everyone will be happy with the ssd/hdd default behavior!